tests/cli/test_health.py

```diff
@@ -0,0 +1,27 @@
+from unittest import mock
+
+from click.testing import CliRunner
+
+from snuba.cli.health import health
+
+
+def test_check_health() -> None:
```

Comment:
nit: the test cases are really good but they should be separate test cases, not one big one

Reply:
Also separated teset cases in `test_views.py`